opm/common/data/Wells.cpp

```diff
@@ -0,0 +1,70 @@
+/*
+  Copyright 2016 Statoil ASA.
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <opm/common/data/Wells.hpp> 
+
+namespace Opm {
+namespace data {
+
+    production::production( double w, double o, double g ) :
+        wat( w ),
+        oil( o ),
+        gas( g )
+    {}
+
+    production::production( const std::vector< double >& vec, size_t index,
```

Comment:
I first defined it as a free function returning a production, and I'm still going back-and-forth on whether this should exist at all, be a constructor or a free function.